<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnmpTrapWidgets</class>
 <widget class="QWidget" name="SnmpTrapWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>197</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,7,1,1,6">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="lblCommunity">
       <property name="text">
        <string>Community</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cmbCommunity</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblAgentAddress">
       <property name="text">
        <string>Agent address</string>
       </property>
       <property name="buddy">
        <cstring>cmbAgentAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="cmbTrapType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Cold Start</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Warm Start</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Link Down</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Link Up</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Authentication Failure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EGP Neighbor Loss</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Enterprise Specific</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbAgentAddress">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>localhost</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblDataAddress">
       <property name="text">
        <string>Data address</string>
       </property>
       <property name="buddy">
        <cstring>cmbDestAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbDestAddress">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="cmbCommunity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>public</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="cmbEnterprise">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblTrapType">
       <property name="text">
        <string>Trap type</string>
       </property>
       <property name="buddy">
        <cstring>cmbTrapType</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="lblSpecific">
       <property name="text">
        <string>Specific</string>
       </property>
       <property name="buddy">
        <cstring>spinSpecific</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QSpinBox" name="spinSpecific">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="lblEnterprise">
       <property name="text">
        <string>Enterprise </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cmbEnterprise</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblMibOid">
       <property name="text">
        <string>MIB OID</string>
       </property>
       <property name="buddy">
        <cstring>cmbMibOid</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbMibOid">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblMibValue">
       <property name="text">
        <string>MIB value</string>
       </property>
       <property name="buddy">
        <cstring>cmbMibValue</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbMibValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblMibType">
       <property name="text">
        <string>MIB type</string>
       </property>
       <property name="buddy">
        <cstring>cmbMibType</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="cmbMibType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>NULL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>INTEGER</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OCTET STRING</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OBJECT IDENTIFIER</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IP ADDRESS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UNSIGNED32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COUNTER</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GAUGE32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TIMETICKS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OPAQUE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COUNTER64</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
